app/channels/game_channel.rb

```diff
@@ -162,45 +166,48 @@ def attack
   end
 
   def intercept(params)
+    @gameboard.reload
     # params={
     # action: "intercept",
     # unique_card_id: 1,
     # to: 'center_card' | 'current_player'
     # }
 
-    unique_card_id = params['unique_card_id']
-    to = params['to']
+    if @gameboard.intercept_phase? || @gameboard.boss_phase?
```

Comment:
eher a early return mit einer error message, feedback is für die spieler und zum debuggen praktisch:
``` ruby
if !@gameboard.intercept_phase? && !@gameboard.boss_phase?
      PlayerChannel.broadcast_error(current_user, 'You cannot itnercept now, it is #{@gameboard.phase} phase')
     return
end
```